### DISCUSSION TITLE
posts/2024/04/field-essay-46/

### DISCUSSION THREAD

**Discussion by giscus** [residualsun1/Residualsun#48] [General]
# posts/2024/04/field-essay-46/

每年农历三月二十是南村天后的诞辰。临近诞辰，南村天后会被提前抱到不同村子的不同家户中轮流祭拜。根据今年负责随天后流转各个村子的大叔所言，流转的顺序由二十多位祖头与钱头在庙宇中决定，而提前祭拜的时间并不固定。1今年，西埠村的祭拜时间在农历三月初一，即今天。

https://guozheng.rbind.io/posts/2024/04/field-essay-46/

**Comment by ghost** [comment]
津前 大殿天后、真衣天后、樂興天后
硇洲十八座行相天后宫，按照各军主之间“金玉满堂”顺序的排序，金字当头。
赤马 金身天后(大婆、金身婆)
津前 金身天后(大婆、金身婆)
大林 玉相天后(二婆) （谭北跨孟岗、南港、北赛、宋皇、新村仔等）
六竹 滿嗣天后(六竹婆) （六竹、晏庭片跨南港（咸宝、谭杰、祝彩））
南村 南村天后(南村婆、亦称堂上天后) 
塘尾 白沙天后(白沙婆)
北金 陳七娘娘(北金婆) 
大伦 博度二相天后(那甘婆) 
谭北湖 潭泰天后、景樂天后(谭北湖两婆) 
谭井 潭泰天后(谭井婆) 
庄屋 北海天后(庄屋婆) 
后角 北海天后(后角婆) 
大浪 北汛天后(大浪婆)
讯地仔 海晏天后(海晏婆) 
橧棚 福安天后(福安婆) 
存亮村天后(堂号未知)
存亮村天后堂号未知，作者可询问当地庙祝、村民、道士了解情况，该天后在存亮村水仙宫庙宇坐镇配祀，预测应该叫南豐天后（猜测）

**Reply by residualsun1** [reply]
哇！！这些信息好详细！！非常感谢😄！！

冒昧一问，请问您是硇洲本地人吗🫣，或者在硇洲做过相关调查呀？(如果您觉得不方便的话不必回答(●'◡'●)）

**Reply by ghost** [reply]
我是本地人，但是我比较少回家，就是我之前看过哪些做调研的学者，就感觉他们所做的学术研究不够透彻，不够清楚、有些不是很正确，其实我也是了解点滴而已，如果你想知道其它问题，我知道的其实也可以告诉你的，你们可以向村委会或者是政府部门寻找下《全粤乡情》、《湛江开发区自然村庄简史》，这两本书里面有很多介绍，介绍各村历史、由来、地理、文化习俗，也许对你们做调研有所帮助，我也想看看《湛江开发区自然村庄简史》这本书，主要是找不到，哈哈哈哈哈哈，只有镇府部门和村委会里面可以找到这些书籍。
最近其实有很多神诞和做戏，其中就包括广福、广德、高岗公、天后等，最近广福、广德会吃斋、打朝，每年农历三月廿八会过衙了，听网络消息说福公回谭北湾村、德公回北赛村。
如果你想要了解硇洲宫庙、神灵、历史，你可以去当地各村的庙堂，硇洲的庙宇都是可以免费进去的，上次我回去做清明就去了通天石那里，那里有座高岗庙。硇洲津前天后宫、赤马天后宫、六竹滿嗣宫、南村天后宫、塘尾天后宫、北金天后宫、谭井天后宫、庄屋天后宫、庄屋天后宫、存亮水仙宫，这几个天后是有庙宇的，其它天后没有神庙。
我上次看了你写文章，你是想了解存亮村水仙宫谭五郎的由来吗，存亮村水仙宫供奉的是埠主譚伍郎两位水仙大王，相传谭五郎原是硇洲岛南港村委会锦马村人士。我们红卫社区水仙宫供奉的境主温州茅竹水仙靈應大王为伟大的爱国诗人屈原。
茅竹水仙有大禹、项羽、屈原、伍子胥、鲁班、李白、王勃、奡、冥、伯益等十人。
其实我也有一个问题想问的，就是西埠村所供奉的是興利埠主茅竹水仙大王，诞辰则为农历正月廿六，西埠村所供奉水仙大王为哪位古人？

**Reply by residualsun1** [reply]
> 我是本地人，但是我比较少回家，就是我之前看过哪些做调研的学者，就感觉他们所做的学术研究不够透彻，不够清楚、有些不是很正确

我认为至少就人类学的研究而言，取得当地人的认可和肯定相当重要。人类学做的田野调查应当是为当地人讲述关于他们自己的故事。

> 如果你想知道其它问题，我知道的其实也可以告诉你的

好呀！之后若有疑问，就麻烦您不吝赐教了😊.

> 你们可以向村委会或者是政府部门寻找下《全粤乡情》、《湛江开发区自然村庄简史》，这两本书里面有很多介绍，介绍各村历史、由来、地理、文化习俗，也许对你们做调研有所帮助

这个建议很有帮助！随着对村庄神明事务的了解，我对各村的历史也越来越好奇了。特别是你还提到一些神明原来是某村的人士，这我从前一直都没有了解。

> 我上次看了你写文章，你是想了解存亮村水仙宫谭五郎的由来吗，存亮村水仙宫供奉的是埠主譚伍郎两位水仙大王，相传谭五郎原是硇洲岛南港村委会锦马村人士。我们红卫社区水仙宫供奉的境主温州茅竹水仙靈應大王为伟大的爱国诗人屈原。
茅竹水仙有大禹、项羽、屈原、伍子胥、鲁班、李白、王勃、奡、冥、伯益等十人。
其实我也有一个问题想问的，就是西埠村所供奉的是興利埠主茅竹水仙大王，诞辰则为农历正月廿六，西埠村所供奉水仙大王为哪位古人？

是的是的，我之前有简单了解到西埠村水仙大王的故事，所以会很好奇谭五郎。你说到茅竹水仙大王都是以历史人物为原型，这个让我大吃一惊。目前我所了解的西埠村水仙大王，几位村民都说是在海上漂到村子的香炉，或者是说他以前抓鱼很厉害，抓的还是鲸鱼，又有“抓鱼祖”一说，但仅此而言，还没见谁说过他是历史上的哪个具体人物，所以你最后的问题我暂时还没办法解答。

另，你所了解的信息其实还挺系统（比如天后的分类）和丰富的（水仙大王的各原型），请问你是咋了解的呀？是读过了《全粤乡情》吗？或者是总结了学者们的调研吗？

**Reply by ghost** [reply]
是的，我是通过《全粤乡情》了写硇洲岛的习俗文化和生活方式，里面对每个村庄都作了很详细的介绍。因为西埠村的水仙冠于茅竹水仙的称号，首先就代表他不属于本土神灵，这里的茅竹指的是一个地方，是指温州茅竹吕大殿，网络上可以查询得到相关消息，其次，有些村民是不知道神灵名号，也许在代代相传的口中已经失传了，你唯一可以了解西埠茅竹水仙名字叫什么只能询问当地的道士了，只有他们才会知道这些神灵代表着啥，名字叫什么。

**Reply by residualsun1** [reply]
> 是的，我是通过《全粤乡情》了写硇洲岛的习俗文化和生活方式，里面对每个村庄都作了很详细的介绍。

我查了一下，似乎应该是《[全粤村情](http://dfz.gd.gov.cn/index/ztlm/szfzg/gctj/dqwx/content/post_3057928.html)》，不过在网站上没找到关于硇洲的详细信息，可能还是要到政府机关那里获取。

> 因为西埠村的水仙冠于茅竹水仙的称号，首先就代表他不属于本土神灵，这里的茅竹指的是一个地方，是指温州茅竹吕大殿，网络上可以查询得到相关消息

感觉这个说法很神奇诶，没想到《全粤村情》竟然可以记载得那么详细，不知道是怎样的一群人以怎样的一种方式进行调查。

> 有些村民是不知道神灵名号，也许在代代相传的口中已经失传了，你唯一可以了解西埠茅竹水仙名字叫什么只能询问当地的道士了，只有他们才会知道这些神灵代表着啥，名字叫什么。

是的，我在村中认识的老人大多数都说不上来由了，只能叫俗称，倒是道士还知道，不过当时没有想到这个问题……

**Reply by ghost** [reply]
广东省人民政府地方志办公室推出全粤村情平台相关数据产品、《全粤村情》、《广东省精准扶贫村情集成》、结构化丛书等系列普查成果。

**Reply by residualsun1** [reply]
> 广东省人民政府地方志办公室推出全粤村情平台相关数据产品、《全粤村情》、《广东省精准扶贫村情集成》、结构化丛书等系列普查成果。

请问你是在政府那儿找到的书吗？我在政府官网只能找到书名。

另，我们最近准备做一个渔民座谈会，会找道士还有部分村民一起讨论收集的资料以及我们的疑问。

**Reply by ghost** [reply]
我不是在政府那里找到的书籍，我当时是去广州图书馆看书的时候看过的，图书馆和地方志馆都有，因为还没有量产，能遇到寻找机会比较少。渔民座谈会挺好的，也许举办完这个座谈会会开启自己又一个新事物认知了，那你们不需要回学校准备毕业论文和答辩的事宜的吗？

**Reply by residualsun1** [reply]
开完渔民座谈会了，还在回顾中……大家讲了不少内容，但囿于语言的不熟悉，纪要由一位参会的一位成员整理了。

这里只有我是学生，但我还在读大三，明年才需要准备毕业论文和答辩。
